apps/family-public-law/fpl-case-service/fpl-case-service.yaml

```diff
@@ -8,12 +8,19 @@ spec:
   releaseName: fpl-case-service
   values:
     java:
-      replicas: 2
+      replicas: 4
+      memoryRequests: '1536Mi'
+      cpuRequests: '250m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '2000m'
```

Comment:
How come CPU limits have been increased from 1500m to 2000m? The usage in prod doesn't appear to justify this increase..